src/components/DryRunnerBrawlModifiers/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Dialog from '../Dialog'
+
+export default React.memo(function DryRunnerBrawlModifiers(props) {
+  const dialogRef = React.useRef()
+  const modifierRef = React.useRef()
+
+  return (
+    <>
+      <label htmlFor='modifier'>Reset with modifier</label>
+      <select
+        name='modifier'
+        id='modifier'
+        data-testid='brawl-modifier'
+        className='DryRunnerBrawlModifiers'
+        value={props.modifier}
+        onChange={event => {
+          modifierRef.current = event.target.value
+          dialogRef.current.show()
+        }}
+      >
+        <option value='0'>None</option>
+        <option value='1'>Structures = 2 mana</option>
+        <option value='2'>Toads = 2 mana</option>
+        <option value='3'>Knights -2 mana</option>
+        <option value='4'>Dwarves -2 mana</option>
+        <option value='5'>Spells -2 mana</option>
+      </select>
+      <Dialog
+        id='brawl-modifiers-dialog'
+        title='Brawl mode'
+        dialogRef={instance => (dialogRef.current = instance)}
+        image='/assets/images/cards/execution.png'
```

Comment:
As a nice touch, if you want, we could use a different image depending on the chosen Brawl modifier. For instance, 2 mana toads would display Tode. :)